_elements/inputs.md

```diff
@@ -19,19 +19,25 @@ title: Inputs
 
 <div class="preview">
 
+  <h2>Text Input</h2>
+
   <label for="input-type-text">Text Input Label</label>
   <input id="input-type-text" name="input-type-text" type="text">
 
   <label for="input-focus">Text Input Focused</label>
   <input class="usa-input-focus" id="input-focus" name="input-focus" type="text">
 
-  <label for="input-error">Text Input Error</label>
-  <input class="usa-input-error" id="input-error" name="input-error" type="text">
+  <div class="input-error">
+    <label for="input-error">Text Input Error</label>
+    <span class="input-error_message" role="alert">Helpful error message</span>
```

Comment:
Shouldn't it be `input-error-message` (with a dash) since it's a different element below `input-error`?